invenio_records_resources/services/records/results.py

```diff
@@ -264,6 +264,14 @@ def get_value_service(self, value):
         """Return the value and the service to fetch the referenced record."""
         return None, None
 
+    @abstractmethod
+    def tombstone(self, value):
+        """Return the tombstone representation of the value.
+
+        This is used to be returned when a value cannot be resolved
+        """
+        return {}
```

Comment:
derivate from the file comment: If it is an `abstractmethod` the inheriting class is forced to reimplement so it does not make sense to have a return value. Alternatively, if we want to have a default (e.g. `{}`) then the decorator should be removed.
```suggestion
     pass
```

Example:

```python
>>> class A(ABC):
...     @abstractmethod
...     def test(self):
...             return {}
...
>>> class B(A):
...     pass
...
>>> B().test()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: Can't instantiate abstract class B with abstract method test
```